lib/buildmodel.js

```diff
@@ -35,14 +35,10 @@ function fetchPipelineId(datastore, jobId, callback) {
  *                                        keys: "pipelineID" and "scmUrl"
  */
 function fetchScmUrl(datastore, params, callback) {
```

Comment:
Are there any other cases when we would need to `fetchScmUrl`? If so, should we consider making it a function provided by the model?